### DISCUSSION TITLE
Proper way to setup misskey for private use

### DISCUSSION THREAD

**Discussion by DeMiro5001** [misskey-dev/misskey#8600] [Q&A]
Hi all, 

I'm happy to discover misskey and start using it. 

I'm trying to setup a new instance of misskey for private use, so no federation required and all what's published must remain private. 

Why do I need this? 

My daughter asked me days ago to have a Facebook account since all her friends have one and wanted to stay connected with them. The idea of letting her have a Facebook account doesn't really please me. So I decided to search for a self-hosted alternative. Misskey was the what I chose. 

But I am struggling to make it correctly configured for this purpose. Making a private space for my kid and her friends to exchange and talk (even though there is no video call for now in misskey). 

Another thing, I can't figure out how to make comments of one user show in the timeline of another user different from the one who shared the note. (in the Facebook style not the twitter one). 

I found a misskey app called "milktea" in the play store, it's good but I think I will use the webapp. 

Finally, thanks to all the Devs and anyone who have made this excellent app possible ❤️

(by the way, I tried to submit this discussion in the misskey forum but couldn't, because of permission and couldn't sign up using github account)

**Comment by ThatOneCalculator** [comment]
I don't think Misskey is what you're looking for, then. Misskey at its core is about a Twitter-like microblogging experience ***with*** federation. I'd probably reccomend [diaspora*](https://diasporafoundation.org/) or [HumHub](https://www.humhub.com/en) for your usage, maybe even a Discord or [Revolt](https://revolt.chat) server or a private XMPP/Jabber MUC (multi-user chat).

**Reply by ThatOneCalculator** [reply]
cc @jarod5001

**Comment by acid-chicken** [comment]
Limited Federation/No Federation mode is planned at #2318 and implementation work in progress at #7709.
I hope Misskey also fits properly your use case.

**Reply by DeMiro5001** [reply]
Thanks a lot. That's a great news. My daughter is 10 years old. She liked misskey and mattermost (I showed her a long list of apps : diaspora, pleroma, movim, hubzilla, humhub, friendica, rocketchat, manyverse and may be others), but she asked me about voice messages and not only voice call, so I may associate misskey and nextcloud talk.
I have set privacy in misskey : reject crawler indexing:on/default visibility : home/local only:on. 
For now I will set my server behind yunohost sso to restrict "visitors" access.
